public/i18n/en.json

```diff
@@ -249,7 +249,8 @@
             "disabledByPolicy": "Feature disabled by administrator",
             "syncFailed": "Group synchronization failed",
             "noMembers": "No members defined",
-            "alreadyLeft": "You have already left this group"
+            "alreadyLeft": "You have already left this group",
+            "valueTooLong": "A value you entered is too long"
```

Comment:
In hindsight, I think this should have been just `invalidValue` because it is so generic anyway...

Reply:
If we wanted it generic we could have used `badRequest`. But I think `valueTooLong` is fine :slightly_smiling_face: